internal/cmd/controller/gitrepo/targetsyaml.go

```diff
@@ -12,6 +12,9 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// XXX: move this to internal/cmd/controller/target?
+// Or reconsider the idea of rebuilding job specs altogether?
+
```

Comment:
```suggestion
```

If we want to be super correct, since this is now used by two different binaries, it should live in `internal/`.
But I already ignored the structure when I created the CLI for `target` and didn't move `internal/cmd/controller/target`

Reply:
Yes, I wrote this comment when I thought that we might be building job specs from 2 different places, which is of course not necessary. Comment removed.